libnmstate/state.py

```diff
@@ -432,6 +433,27 @@ def remove_unknown_interfaces(self):
         for unknown_iface in unknown_ifaces:
             del self.interfaces[unknown_iface]
 
+    def _route_is_next_hop_to_auto_iface(self, route):
```

Comment:
Can we use `dynamic_ip_iface` instead of `auto_iface`?
`auto` may refer also to `auto-negotiation`.